File: fizzbuzz/fizzbuzz.js
var fizzbuzz = function(start, final){
  fizzbuzz = ""
  for (i = start; i < (final + 1); i++){
    if (i % 15 === 0)
      fizzbuzz += " fizzbuzz"
    else if (i % 3 === 0)
      fizzbuzz += " fizz"
    else if (i % 5 === 0)
      fizzbuzz += " buzz"
    else
      fizzbuzz += " " + i
  }
  return fizzbuzz
}

console.log(fizzbuzz(1,100))

File: fizzbuzz/fizzbuzz.rb
def fizzbuzz(start, final)
  array = (start..final).to_a
  fizzbuzz_str = ""

  array.each do |elem|
    if (elem % 15).zero?
      fizzbuzz_str += " fizzbuzz"
    elsif (elem % 3).zero?
      fizzbuzz_str += " fizz"
    elsif (elem % 5).zero?
      fizzbuzz_str += " buzz"
    else
      fizzbuzz_str += " #{elem.to_s}"
    end
  end
  puts fizzbuzz_str
end

fizzbuzz(1,100)
